easybuild/easyconfigs/m/mDust/mDust-20231029-GCC-10.3.0.eb

```
@@ -0,0 +1,37 @@
+easyblock = 'MakeCp'
+
+name = 'mDust'
+version = '20231029'
+
+homepage = 'https://github.com/lh3/mdust'
+description = "mdust from DFCI Gene Indices Software Tools (archived for a historical record only)"
+
+toolchain = {'name': 'GCC', 'version': '10.3.0'}
+
+sources = [{
+    'filename': 'mdust.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/lh3',
+        'repo_name': 'mdust',
+        'commit': '3e3fed8',
+        'clone_into': 'mdustSrc',
+    },
+}]
+# git clone, might have to adjust this checksum
+checksums = ['6fe21806e6bcd3810babed8d83dfb9ff072fcb9d3a31cd43da9cb9670c0ec6f0']
```

Since this can change, we usually set the checksum to `None`:
```suggestion
checksums = [None]
```